[THREESCALE-1150] Add option to use only path-based routing (#1035)
3scale/APIcast: doc/parameters.md

When this parameter is set to _true_, the gateway will use path-based routing in addition to the default host-based routing. The API request will be routed to the first service that has a matching mapping rule, from the list of services for which the value of the `Host` header of the request matches the _Public Base URL_.

### `APICAST_PATH_ROUTING_ONLY`

**Values:**
- `true` or `1` for _true_
- `false`, `0` or empty for _false_

> When this parameter is set to _true_, the gateway will use path-based routing and will not fallback to the default host-based routing. The API request will be routed to the first service that has a matching mapping rule, from the list of services for which the value of the `Host` header of the request matches the _Public Base URL_.

Reviewer: ```suggestion
When this parameter is set to _true_, the gateway uses path-based routing and does not fallback to the default host-based routing. The API request is routed to the first service that has a matching mapping rule, from the list of services for which the value of the `Host` header of the request matches the _Public Base URL_.
```

Author: Changed :+1: 
Thanks for the review.
